package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gpii-electron",
+  "version": "1.0.0",
+  "description": "Electron application and libraries to run the GPII as a native user space application on the desktop",
+  "main": "main.js",
+  "scripts": {
+    "start": "electron main.js",
+    "test": "electron tests.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gpii/gpii-electron.git"
+  },
+  "keywords": [
+    "Electron",
+    "quick",
+    "start",
+    "tutorial"
+  ],
+  "devDependencies": {
+    "electron": "1.4.1",
+    "request": "^2.69.0",
+    "shelljs": "0.3.0"
+  },
+  "dependencies": {
+    "infusion": "git://github.com/fluid-project/infusion.git#99765614ca5679580024e8b673189ed419468a0c",
+    "windows": "git://github.com/javihernandez/windows.git#6a1d8fdb0836c6ff3aa9b355fa83ae8938680558",
```

Comment:
Are there particular fixes you require from these branches? It would be helpful to add a README explaining which branch they are bound to and why we need them